core/src/test/java/org/apache/iceberg/rest/responses/TestFetchScanTasksResponseParser.java

```diff
@@ -95,7 +95,7 @@ public void roundTripSerdeWithDeleteFilesNoFileScanTasksPresent() {
 
     String invalidJson =
         "{\"plan-tasks\":[\"task1\",\"task2\"],"
-            + "\"delete-files\":[{\"spec-id\":0,\"content\":\"POSITION_DELETES\","
+            + "\"delete-files\":[{\"spec-id\":0,\"content\":\"position-deletes\","
             + "\"file-path\":\"/path/to/data-a-deletes.parquet\",\"file-format\":\"PARQUET\","
```

Comment:
it seems like file-format is also lower-case : 

https://github.com/apache/iceberg/blob/35d66a3fe8ad76e1c76b39a898d9477f1da023ae/open-api/rest-catalog-open-api.yaml#L4329-L4335